### DISCUSSION TITLE
What animals featured on the 🇦🇺 coat of arms?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#792] [Galaxy Trivia] [answered]
**Bonus:** What do the animals in the original coat symbolize?

**Comment by peterbaccaro** [comment, marked as answer]
The coat of arms of Australia features two iconic Australian animals: the **kangaroo** and the **emu**. The kangaroo is on the left side of the coat of arms, and the emu is on the right side. These animals were chosen to **symbolize progress** because they are both unable to walk backward, signifying Australia's desire to always move forward as a nation.

**Comment by SANKARAMDAS** [comment]
the kangaroo and the emu

The coat of arms consists of a shield, containing the six official state symbols, which is held up by a kangaroo and an emu, These animals were chosen to symbolize progress because they are both unable to walk backwards. There is also the official seven-pointed Commonwealth Star above the shield.

Each part of the Australian Coat of Arms has its own individual meaning which contributes to the Australian Coat of Arms' symbolism as a whole. It's very common for countries to use symbols on their flags and coats of arms. Symbols help convey a sense of national identity and often reference a country's history. In other words, symbols allow you to tell a national story.

**Comment by ParanoidUser** [comment]
Are there any other animals besides 🦘 and :ostrich: on the coat? 🙄

**Reply by SANKARAMDAS** [reply]
* Lion(Yellow) 
* Red lion passant

![Coat_of_Arms_of_Australia svg](https://github.com/ParanoidUser/yolo/assets/31897843/b2ed0c3a-1150-41b1-919b-e1ac5e769d15)


Bird Catagory: 
the Australian piping shrike
a black swan

**Reply by SANKARAMDAS** [reply]
And it is an emu. Emu is the second-tallest living bird after its ratite relative the ostrich.

**Reply by peterbaccaro** [reply]
In addition to the **kangaroo** and the **emu**,  the central shield features the **lion** and stars of New South Wales, the **black swan** of Western Australia, the walking **lion** of Tasmania and the **piping shrike** of South Australia.
